app/models/mdm/web_page.rb

```diff
@@ -1,4 +1,5 @@
 # Web page requested from a {#web_site}.
+require 'webrick'
```

Comment:
I only see `Webrick::Cookie` being used in the tests, why is this `require` necessary in production or development as being here would imply?  If you only need the require for testing, then put it in the spec only.